std/algorithm/searching.d

```diff
@@ -2330,6 +2330,30 @@ if (isForwardRange!R1 && isForwardRange!R2
     assert(findSkip(s, "def") && s.empty);
 }
 
+struct FindSplitResult(R1, R2) if (isForwardRange!R1 &&
```

Comment:
Make these guys Voldemort types inside the respective functions. In the documentation mention "returns a subtype of `Tuple!(...)` with opCast defined for bool" etc. Then we should be good to go.